src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__list">
```

Comment:
implements these requirement points
<img width="739" alt="image" src="https://user-images.githubusercontent.com/102411130/233562542-809d15f9-7627-46e0-84b3-852467aafc86.png">

if you have any question, just feel free to ask in `fe_chat`